src/bgen/BindingTouch.cs

```diff
@@ -372,6 +373,11 @@ int Main3 (string [] args)
 		return 0;
 	}
 
+	public bool IsApiAssembly (Assembly assembly)
+	{
+		return assembly == apiAssembly;
+	}
+
```

Comment:
Can we move this to the Generator class? That way we do not need to call it as BindingTouch.IsApiAssembly, just call the method. Ideally would be nice to be able to have the generator be self-contained and that way be able to unit test things like this.

Would be nice to have the integration tests we have atm + unit tests for this kind of logic.

Reply:
@mandel-macaque fixed